components/dashboard/src/user-settings/Integrations.tsx

```diff
@@ -270,7 +270,7 @@ function GitProviders() {
             case "api":
                 return "Allow making API calls (used to set up a webhook when enabling prebuilds for a repository)";
             case "read_repository":
-                return "Read/write access to your repositories";
```

Comment:
I'm confused. I thought this scope is indeed about read and write access although it is only named `read_repository` cc @AlexTugarev

Reply:
Hm, then maybe I'm confused as to how we want to resolve this?

My understanding was we wanted to bring our documentation+copy in line with what GitLab says. This is what I can tell from: https://linear.app/gitpod/issue/ENT-176/improve-doc-around-integrating-gitlab-with-enterprise.

If that's not the case, let's update the task there. :pray: